transformers/senml/transformer.go

```diff
@@ -59,9 +57,10 @@ func (t transformer) Transform(msg messaging.Message) (interface{}, error) {
 	msgs := make([]Message, len(normalized.Records))
 	for i, v := range normalized.Records {
 		// Use reception timestamp if SenML messsage Time is missing
-		t := v.Time
-		if t == 0 {
-			t = float64(time.Now().UnixNano())
+		time := v.Time
```

Comment:
I would avoid the name `time`, as it relates to Go library name... I prefer `t` in this case.